common/dynamicconfig/shared_constants.go

```diff
@@ -36,6 +36,27 @@ var defaultNumTaskQueuePartitions = []ConstrainedValue{
 		Value: 1,
 	},
 
+	// The system activity worker task queues in all namespaces should only have one partition, since
+	// we'll only run one worker per task queue.
```

Comment:
these are all only in `primitives.SystemLocalNamespace`, right? there should also be a namespace constraint too